src/v/cloud_storage/tests/segment_meta_cstore_fuzz.cc

```diff
@@ -463,11 +463,27 @@ struct fmt::formatter<cstore_operation>
               []<typename OP>(OP const& op) {
                   if constexpr (reflection::arity<OP>() == 0) {
                       return fmt::format("{}", serde::type_str<OP>());
-                  } else {
+                  } else if constexpr (reflection::arity<OP>() == 1) {
+                      return fmt::format(
+                        "{}({})",
```

Comment:
I don't know if this format change is intentional, it gained brackets and lost spaces after commas.

Reply:
yeh i think the original formatting of std::tuple was (a,b,c)

Reply:
> yeh i think the original formatting of std::tuple was (a,b,c)

Oh, I somehow [missed the parentheses](https://fmt.dev/9.1.0/api.html#range-and-tuple-formatting)